patchwork-user-profiles.md

```diff
@@ -1,299 +0,0 @@
-# Patchwork: User profiles
```

Comment:
right! Thanks @ahdinosaur now i understand.
I agree this is valuable - but should it be in with the api docs?